core/storage/topic/gae_models.py

```diff
@@ -405,6 +405,10 @@ class TopicRightsModel(base_models.VersionedModel):
 
     # The user_ids of the managers of this topic.
     manager_ids = ndb.StringProperty(indexed=True, repeated=True)
+
+    # The user_ids of users who are (or were in history) members of manager_ids.
```

Comment:
Why is this still here? Please make sure to proofread the PR diff carefully before requesting review.

Reply:
Removed, I have proofread this PR but missed this :(